### PR TITLE
Make it clear that git status is introduced in Episode 3 (issue #685)

### PR DESCRIPTION
**Pull request by jdvorak001**
I am suggesting this fix to issue #685.

### CODE REVIEW

**Comment by munkm** [comment]
Hey @jdvorak001 thanks for the review request! I'm sorry it's taken a bit to get to your PR. I'll do my best to get to it this week!

**Review by kekoziar — COMMENTED** [review]
I don't think that starting with explaining nested repositories is the best introduction to Git for new learners. It seems like they should first learn out to initialize a repository and what that means, then a little later have them do a thought experiment as to what should be made into repositories.

**Comment by jdvorak001** [comment]
Thanks for the review. It certainly is not the intent of this path to explain nested repositories, let alone promote them. It just warns against them (as does the currently released version). What I tried to plant is the good practice of running `git status` before `git init`. With that, the `git status` command is introduced a little.

File: _episodes/03-create.md [hunk 1]
```diff
@@ -44,21 +47,36 @@ $ cd planets
 {: .language-bash}
 
 Then we tell Git to make `planets` a [repository]({{ page.root }}{% link reference.md %}#repository)
--- a place where Git can store versions of our files:
+-- a place where Git can store versions of our files.
 
+It is good practice to first check if this is a good place to do so:
+a new repository shall not be nested in another repository placed higher up the directory hierarchy.
```
Comment (kekoziar) [inline _episodes/03-create.md:53]: this sentence introduces topic of nested repositories. Learners will ask, "what does that mean" and "why shouldn't we do that" which is explained later in the lesson.

**Review by kekoziar — COMMENTED** [review]
This PR basically moves the "Places to Create Git Repositories" exercise into the lesson prior to learners first initializing a repository.  I don't agree with the change because learners do not have the framework of initializing repositories on which to add the concept of nested repositories.

**Comment by kekoziar** [comment]
I'm going to close this PR in preparation of the new workbench lesson architecture, but with the following points which halted the original merging of this PR.

PRs should be limited to one issue per PR.  This PR attempted to do many things, and so the original issue - introducing `git status` at the very beginning - was lost and never implemented.  I happen to tell learners that if they need to remember only one thing, it's `git status`. I absolutely agree with and approve moving it to prior to `git init` with explanation.  However, there were questions about the other content that was added to the PR.  Please submit issues as separate PRs in the future.

**Pull request closed without merging**